ve/openmp/CMakeLists.txt

```diff
@@ -95,7 +95,8 @@ endif()
 # Let the user overwrite the compile command
 set(VE_OPENMP_COMPILER_CMD "${CMAKE_C_COMPILER}"                             CACHE STRING "JIT-Compiler")
 set(VE_OPENMP_COMPILER_FLG "${VE_OPENMP_COMPILER_FLG}"                       CACHE STRING "JIT-Compiler flags")
-set(VE_OPENMP_COMPILER_INC "-I${CMAKE_INSTALL_PREFIX}/share/bohrium/include" PARENT_SCOPE)
+set(VE_OPENMP_COMPILER_INC_DIR "${CMAKE_INSTALL_PREFIX}/share/bohrium/include/"  PARENT_SCOPE)
+set(VE_OPENMP_COMPILER_INC     "-I${CMAKE_INSTALL_PREFIX}/share/bohrium/include" PARENT_SCOPE)
```

Comment:
Can we remove this line?

Reply:
Not this line per se, as the `OpenMP` engine still uses it, but I have made another commit, that cleans this up a bit.